velox/functions/CoverageUtil.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+#include <unordered_set>
+
+void printCoverageMapForAll(const std::string& domain = "");
+
+void printVeloxFunctions(
+    const std::unordered_set<std::string>& kLinkBlockList,
+    const std::string& domain = "");
+
+void printCoverageMapForMostUsed();
```

Comment:
Should this function take domain argument as well?

Reply:
You are right.